plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/fetch/FetchPresenter.java

```diff
@@ -78,20 +78,20 @@ public void showDialog() {
         project = appContext.getCurrentProject();
         view.setRemoveDeleteRefs(false);
         view.setFetchAllBranches(true);
-        getRemotes();
+        setRemotes();
     }
 
     /**
-     * Get the list of remote repositories for local one. If remote repositories are found, then get the list of branches (remote and
+     * Set the list of remote repositories for local one. If remote repositories are found, then set the list of branches (remote and
      * local).
      */
-    private void getRemotes() {
+    private void setRemotes() {
```

Comment:
Why do you choose setRemotes name for this method. In general method with name set should have some parameters which it will be set. I think this method should have name something like get or fetch or update, but not set. @vinokurig WDYT?